docs/_layouts/default.html

```diff
@@ -62,6 +62,36 @@
 
   <div id="fb-root"></div>
   <script>
+    (function() {
```

Comment:
It seems inconsistent to inline this when things like `live_editor.js` are included as separate scripts, what do you think about refactoring this out similar to the other stuff?

I suspect people in the future will be digging through old code and trying to figure out what the heck this bit is doing, so would you mind adding just enough overall comments to explain what it is that this section of code is trying to accomplish?